src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -0,0 +1,174 @@
+                                    <th>Exit code</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {% for type, tasks in failed_tasks.items %}
+                                    <tr>
+                                        <td colspan="3" style="background-color: #aaa; font-weight: bold;">
+                                            {{ type }} (total = {{ tasks.count }})
+                                        </td>
+                                    </tr>
+                                    {% for item in tasks %}
+                                        <tr>
+                                            <td>{{ item.filename }}</td>
+                                            <td>{{ item.fileuuid }}</td>
+                                            <td>{{ item.exitcode }}</td>
+                                        </tr>
+                                    {% endfor %}
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+            <table class="footer-wrap">
+                <tr>
+                    <td></td>
+                    <td class="container">
+                        <div class="content">
+                            <table>
+                                <tr>
+                                    <td align="center">
+                                        <p>Archivematica</p>
+                                    </td>
+                                </tr>
+                            </table>
+                        </div>
+                    </td>
+                    <td></td>
+                </tr>
+            </table>
+        </body>
+    </html>
+"""
+
+
+def report(uuid):
+    """
+    Generate normalization report using Django's template module and send it to
+    every active user.
+    """
+    recipient_list = User.objects.filter(is_active=True).values_list('email', flat=True).exclude(email__in=['demo@example.com', ''])
+    if not recipient_list:
+        logger.error('Normalization report is not being sent because the recipient list is empty.')
+        return 0
+
+    logger.info('Sending report to %s.', ', '.join(recipient_list))
+
+    try:
+        sip = SIP.objects.get(uuid=uuid)
+    except SIP.DoesNotExist:
+        logger.error('SIP with UUID %s not found.', uuid)
+        return 1
+
+    failed_tasks = {}
+    for jobtype in ('Normalize for preservation', 'Normalize for access'):
+        try:
+            job = Job.objects.filter(sipuuid=uuid, jobtype=jobtype).order_by('-createdtime')[0]
+        except IndexError:
+            logger.info('No normalization failures have been detected in type "%s"', jobtype)
+            continue
+        tasks = Task.objects.filter(Q(job=job) & ~Q(exitcode=0))
```

Comment:
I think this is the same as `Task.objects.filter(job=job).exclude(exitcode=0)` since `exclude` is filter-not